storybook/src/components/Screen/Screen.docs.mdx

```diff
@@ -8,6 +8,8 @@ import README from "../../../../packages/css/src/components/screen/README.md?raw
 
 <Primary />
 
+<Controls />
+
```

Comment:
Grid is missing Controls as well and I’m finding some others if I search for `import { Canvas, M`.

Reply:
I think the others are all components without props or complicated components which don't have obvious controls (like `Grid`). `Screen` is a simple component with obvious controls, which is why I added them. 

I think we have to think about how we want to document the more complicated components.

Reply:
Discussed and added them for `Grid` and `Header`.